manifests/daemon.pp

```diff
@@ -72,6 +72,7 @@
   Stdlib::Ensure::Service $service_ensure                    = 'running',
   Boolean $service_enable                                    = true,
   Boolean $manage_service                                    = true,
+  Boolean $env_vars_sensitive                                = false,
```

Comment:
please add the documentation for the new parameter at the top

Reply:
Hi @bastelfreak
I intended to do that (already added in elasticsearch_exporter.pp). However,  there are even no documentation for the related parameter (env_vars and env_file_path). But anyway, I'll add doc for all those 3 params.